runtime/doc/eval.txt

```diff
@@ -7641,9 +7641,9 @@ range({expr} [, {max} [, {stride}]])				*range()*
 <
 
 rand([{expr}])						*rand()*
-		Return a pseudo-random Number generated with an xorshift
-		algorithm using seed {expr}.  The returned number is 32 bits,
-		also on 64 bits systems, for consistency.
+		Return a pseudo-random Number generated with an
+		xoroshiro128** algorithm using seed {expr}.  The returned number
```

Comment:
Typo?

Reply:
I confused with the previous name of that.
I will fix them.